src/Types/Inputs/EdgeWhereConditions.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mrap\GraphCool\Types\Inputs;
+
+use GraphQL\Type\Definition\InputObjectType;
+use Mrap\GraphCool\Definition\Relation;
+use Mrap\GraphCool\Types\DynamicTypeTrait;
+use Mrap\GraphCool\Types\Type;
+
+class EdgeWhereConditions extends InputObjectType
+{
+
+    use DynamicTypeTrait;
+
+    public static function prefix(): string
+    {
+        return '_';
+    }
+
+    public static function postfix(): string
+    {
+        return 'EdgeWhereConditions';
+    }
+
+    public function __construct(Relation $wrappedType)
+    {
+        parent::__construct([
```

Comment:
muss man da überhaupt den constructor angeben wenn darin nur der parent constructor aufgerufen wird?

Reply:
Ich übergebe dem parent-constructor ja ein array. wo würde das array denn sonst herkommen?

Reply:
kk, war in der typescript welt drinnen :laughing: da werden die ganze zeit configs in die constructer herumgeschoben, 
 speziell hier, weil noch so viel zusätzlich dazukommt in den arguments würds jz natürlich no sinn machen